src/common/clib-cache.c

```diff
@@ -1,3 +1,10 @@
+//
+// clib-cache.c
+//
+// Copyright (c) 2016-2020 clib authors
```

Comment:
@Isty001 you put this together, but since it's been moved into this repo, I assume it's OK we put this copyright on it. LMK what you think

Reply:
Yes, sure!